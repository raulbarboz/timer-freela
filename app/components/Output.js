import React from 'react';

function Output(props){
    return(
        <div className="output">
            <div className="output__box">
                <p>Com Plano:</p>
                <span 
                className="outpu__box--span">
                {
                   props.withPlan
                }
                </span>
            </div>
            <div className="output__box">
                <p>Sem Plano:</p>
                <span className="outpu__box--span">
                {
                    props.withoutPlan
                }
                </span>
            </div>
        </div>
    )
}

export default Output;