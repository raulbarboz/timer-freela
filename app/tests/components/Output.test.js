import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Output from '../../components/Output';

test('should render Output correctly', () => {
    const wrapper = shallow(<Output />);
    expect(toJSON(wrapper)).toMatchSnapshot()
})