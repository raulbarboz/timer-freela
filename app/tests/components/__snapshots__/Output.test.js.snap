// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Output correctly 1`] = `
<div
  className="output"
>
  <div
    className="output__box"
  >
    <p>
      Com Plano:
    </p>
    <span
      className="outpu__box--span"
    />
  </div>
  <div
    className="output__box"
  >
    <p>
      Sem Plano:
    </p>
    <span
      className="outpu__box--span"
    />
  </div>
</div>
`;
